spec/integ/crypto/megolm-backup.spec.ts

```diff
@@ -134,37 +149,42 @@ describe("megolm key backups", function () {
             },
         };
 
-        return aliceTestClient!
-            .start()
-            .then(() => {
-                return createOlmSession(testOlmAccount, aliceTestClient);
-            })
-            .then(() => {
-                const privkey = decodeRecoveryKey(RECOVERY_KEY);
-                return aliceTestClient!.client!.crypto!.storeSessionBackupPrivateKey(privkey);
-            })
-            .then(() => {
-                aliceTestClient!.httpBackend.when("GET", "/sync").respond(200, syncResponse);
-                aliceTestClient!.expectKeyBackupQuery(ROOM_ID, SESSION_ID, 200, CURVE25519_KEY_BACKUP_DATA);
```

Comment:
it would be nice to rip out the now-unused `TestClient.expectKeyBackupQuery`, but maybe that's a job for another day